gnd/src/main/java/com/google/android/gnd/persistence/local/room/entity/FeatureEntity.java

```diff
@@ -120,6 +132,8 @@ public static FeatureEntity fromFeature(Feature feature) {
       entity.setLocation(Coordinates.fromPoint(((PointFeature) feature).getPoint()));
     } else if (feature instanceof GeoJsonFeature) {
       entity.setGeoJson(((GeoJsonFeature) feature).getGeoJsonString());
+    } else if (feature instanceof PolygonFeature) {
+      entity.setPolygonVertices(listToString(((PolygonFeature) feature).getVertices()));
```

Comment:
The method names `listToString` and `stringToList` are not clear within their scope/namespace `FeatureEntity`. To make them non-ambiguous, qualify that these are being used to convert a list of _vertices_ specifically. For example, `FeatureEntity#formatVertices` and `FeatureEntity#parseVertices`.

Reply:
updated